### PR TITLE
PNG files have full-screen checkerboard background & Better emptycontent view for file types which can not be shown

### PR DESCRIPTION
**Pull request by mario**
This branch does the integration with master in a really, really nice way. Please approve 📦 

#456 
#455

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
aaaand the build fails ;)

**Comment by mario** [comment]
@AndyScherzinger checking

**Comment by mario** [comment]
@AndyScherzinger looks better now ;)

**Comment by tobiasKaminsky** [comment]
👍

**Comment by AndyScherzinger** [comment]
@mario @jancborchardt I like the new "corrupt image" optimization, but please bring back the transparent system and toolbar! :) I personally don't want to view my pictures with a blue bar on top that overlays parts of the image, I want the transparent bar back :P

**Comment by mario** [comment]
@AndyScherzinger transparent ones look VERY ugly on the black background. Please click on the image and you lose the top part completely. In the future, I plan to completely ditch the nav/action bar images/videos, but we'll see about that...

**Comment by AndyScherzinger** [comment]
@mario I am sorry...but:

> transparent ones look VERY ugly on the black background.

Nope: you think they do

> I plan to completely ditch the nav/action bar images/videos, but we'll see about that...

So what is your proposed solution then when posting such a "plan"

**Comment by mario** [comment]
@AndyScherzinger yes, because they look ... black :D

b) I dunno yet, that's just an idea - when I'm ready with it, I'll open an issue for discussing.

**Comment by AndyScherzinger** [comment]
The issue is that you "cu off" to top part on an image that fills the whole screen and to removed that hint that the is much to see and need to know that you can hide the bar

**Comment by AndyScherzinger** [comment]
👍 code wise
added the design label for designer's approval for the action bar coloring discussion https://github.com/nextcloud/android/pull/607#issuecomment-277542678 ff.

**Comment by mario** [comment]
@eppfel plese chime in and approve.

**Comment by eppfel** [comment]
Can I have screenshots? Very hard to follow..

**Comment by mario** [comment]
Okay all, this is once again ready for testing and review. I've made it play the same as official Gallery app - no status bar, but translucent navigation bar. Hope you like it!

**Comment by AndyScherzinger** [comment]
@mario can you provide @eppfel with some screenshots? I am not at my laptop...
If there is anything you me to discuss with the designers at Wednesday's meetup,just let me know :)

**Comment by mario** [comment]
Here we go @eppfel:

screenshot_20170214-112305

**Comment by jancborchardt** [comment]
Looks good. Agree with Andy on the transparent bar. :)

**Comment by mario** [comment]
@jancborchardt well, I added it back so there - be happy :P

**Comment by mario** [comment]
@AndyScherzinger this one too ^_^

**Comment by AndyScherzinger** [comment]
@mario sure, but same issue here:

> **Conflicting files**
res/layout/file_preview.xml
res/layout/media_control.xml
res/layout/preview_image_fragment.xml
res/layout/text_file_preview.xml
res/values-en-rGB/strings.xml
res/values/strings.xml
res/values/styles.xml
src/com/owncloud/android/media/MediaControlView.java
src/com/owncloud/android/ui/activity/DrawerActivity.java
src/com/owncloud/android/ui/preview/PreviewImageActivity.java
src/com/owncloud/android/ui/preview/PreviewImageFragment.java
src/com/owncloud/android/ui/preview/PreviewMediaFragment.java
src/com/owncloud/android/ui/preview/PreviewTextFragment.java
src/third_parties/michaelOrtiz/TouchImageViewCustom.java

**Comment by AndyScherzinger** [comment]
@mario can you rebase this one? :) Then I can test drive it 🚀

**Comment by AndyScherzinger** [comment]
👍 @mario nevermind, I merged the latest master and successfully tested it. Please review @tobiasKaminsky  :)

One minor thing, could we get the system bar back and make the bars blue in case of "the image can't be displayed"? If it is too much work I'd say we postpone it but open an issue to not forget it.

device-2017-04-03-233332

**Comment by mario** [comment]
@tobiasKaminsky we need to merge this, so please do it.

**Comment by mario** [comment]
Well to me this looks already approved, but I'll wait for @tobiasKaminsky to do it in this case.

**Comment by tobiasKaminsky** [comment]
👍

**Comment by tobiasKaminsky** [comment]
> One minor thing, could we get the system bar back and make the bars blue in case of "the image can't be displayed"? If it is too much work I'd say we postpone it but open an issue to not forget it.

can you do this? @AndyScherzinger

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky I might, but will only be able to work on this after the vacation, so end of April if that is fine. Could you open an issue and assign it to me? :)
